site/en/community/_toc.yaml

```diff
@@ -14,21 +14,20 @@ toc:
 
 - name: "Contribute"
   contents:
-  - title: "Contribute to TensorFlow"
+  - heading: "Contribute to TensorFlow"
```

Comment:
Headings don't link to things.

Each link needs a "title:"
 
See the [guide _toc](https://github.com/tensorflow/docs/blob/master/site/en/guide/_toc.yaml) for example